### ISSUE TITLE
How can i use this api in my app,pod can not work 

### ISSUE THREAD

**Issue by Fengur** [GoogleCloudPlatform/ios-docs-samples#42]
does "googleapis"  can be userd in a new app, how can i set up podfile to run this service?
so confused about it ,really need some help

**Comment by wasimsafdar** [comment]
Yes, add Google folder and googleapis.podspec in your project root folder. Add "pod 'googleapis', :path => '.' " in your pod file and then update or install pods. However, I am getting 1 crash after that and I raised this issue.

**Comment by hzh137940556** [comment]
@wasimsafdar   hi， my app dont use pods but carthage, I need to manually integration，can you help me?
